### PR TITLE
My first Pull request... 

### PR DESCRIPTION
**Pull request by madhumohan01**
@sNowCat0 Please review the attractions group.

### CODE REVIEW

File: madhumohan01-chennai.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Chennai
+
+### Restaurants
+- Lots
+
+### Attractions
+- Marina
+- Shopping
```
Comment (sNowCat0) [inline madhumohan01-chennai.md]: What kind of shopping? Can I buy a stairway to heaven? Is there food? I'm on a *See* food diet :fish: :sushi:  :wink:

File: madhumohan01-chennai.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Chennai
+
+### Restaurants
+- Lots
```
Comment (sNowCat0) [inline madhumohan01-chennai.md]: Is there a McDonald's?

File: madhumohan01-chennai.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Chennai
```
Comment (sNowCat0) [inline madhumohan01-chennai.md:1]: What time of year is good time to visit? :world_map: :airplane:

**Review by sNowCat0 — CHANGES_REQUESTED** [review]
Add some restaurants please.

**Comment by madhumohan01** [comment]
@sNowCat0 Please review the new changes...

**Comment by madhumohan01** [comment]
@sNowCat0 Please review changes.

**Review by sNowCat0 — APPROVED** [review]
Good to Go!

*Not responsible for production outages :wink:

**Pull request merged**